package/gluon-mesh-babel/check_site.lua

```diff
@@ -1,4 +1,5 @@
-need_string_match(in_domain({'node_client_prefix6'}), '^[%x:]+/64$')
+need_string_match(in_domain({'prefix6'}), '^[%x:]+/64$')
+need_string_match(in_domain({'node_prefix6'}), '^[%x:]+/64$')
```

Comment:
this is already required by layer3-common

Reply:
I think I'd add that anyways, even if a dependency pulls in the same check, too. Tracking if a check was added by another dependency seems tedious. Also if someone were to remove the usage of the site setting for gluon-layer3-common later, there is a high chance that the check would be removed in gluon-layer3-common, too and that the needed check in gluon-mesh-babel would then be missed.

Therefore it seems the most simple and less error prone approach to me, to stick to this simple rule: If this package requires a site/domain setting, then add it to this package's check_site.lua. Even if another dependcy package has it in its check_site.lua as well.